### PR TITLE
net-misc/frr: Version bump to 8.0 

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: dev-util/clippy/clippy-8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="frr-${PV}"
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (SoapGentoo) [inline dev-util/clippy/clippy-8.0.ebuild:7]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```

File: net-misc/frr/frr-8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (SoapGentoo) [inline net-misc/frr/frr-8.0.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```

File: net-misc/frr/frr-8.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+	sys-libs/libcap
+	sys-libs/readline:0=
+	virtual/libcrypt:=
+	grpc? ( net-libs/grpc:= )
+	nhrp? ( net-dns/c-ares:0= )
+	pam? ( sys-libs/pam )
+	rpki? ( >=net-libs/rtrlib-0.6.3[ssh] )
+	snmp? ( net-analyzer/net-snmp:= )
+"
+
```
Comment (SoapGentoo) [inline net-misc/frr/frr-8.0.ebuild:36]: can you remove these interleaving empty lines? the unnecessarily increase vertical whitespace

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-util/clippy/clippy-8.0.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-util/clippy/clippy-8.0.ebuild]: Done

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**